decentralization.adoc

```diff
@@ -314,11 +313,11 @@ interactions or rules of engagement between neighboring fish that make
 it all work out.
 ____
 
-She points out that many systems, in nature or man-made, can and do
-work without leaders, and they are very powerful and resilient. Each individual
+She points out that many systems, natural or artificial, can and do
+work without leaders, and they are robust and resilient. Each individual
```

Comment:
robust and resilient are almost the same thing. Maybe use "and they are powerful and resilient" instead?